Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy.swift

```diff
@@ -350,6 +350,18 @@ struct PathHierarchy {
         return newReference
     }
     
+    mutating func addOverloadGroupChild(parent: ResolvedIdentifier, name: String, kind: String, symbol: SymbolKit.SymbolGraph.Symbol?) -> ResolvedIdentifier {
+        let parent = lookup[parent]!
+        
+        let newReference = ResolvedIdentifier()
+        let newNode = Node(symbol: symbol, name: name)
```

Comment:
It's too late to add symbols to the path hierarchy after it has been initialized. The added symbol won't be able to affect the other symbols.